tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/TestData/Mock_Typed.json

```diff
@@ -1,124 +1,92 @@
 {
-  "query": "fly from seattle to dallas at 3pm on delta",
+  "query": "http://foo.com is where you can get a weather forecast for seattle",
```

Comment:
Why did the query change?

Reply:
The query was modified in #775 ([link](https://github.com/Microsoft/botbuilder-dotnet/pull/775/files#diff-d522bf150179cec840b54638d357c5b8L2)).